Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:18.04
```

Comment:
why not use `alpine` and `apk`?

Reply:
yeah, this was prototyping to allow myself to compile the apk so I won't need to install from the google play and I could skip one trust hop.

I didn't put too much effort into improving this afterwards because I wasn't sure if you guys are even interested. if you are I can do that :)

Reply:
not part of the 🙌 so I can't really approve in their name.  
Just wanted to start contributing by helping with open PRs

Reply:
regardless, alpine is 10 times lighter than ubuntu which is pretty optimal for creating containers